arches/app/templates/views/components/etl_modules/import-single-csv.htm

```diff
@@ -159,6 +159,10 @@ <h4>
     </h4>
     <!-- ko if: load_details -->
     <!-- ko ifnot: loading() -->
+    <div>
+        <span class="etl-loading-metadata-key" data-bind="text: $root.translations.fileName + ':'"></span>
+        <span class="etl-loading-metadata-value" data-bind="text: load_details.file_name"></span>
```

Comment:
`load_details` should be changed to `loadDetails` to be consistent with recent changes in core arches.